src/fixture/fixture.test.ts

```diff
@@ -68,3 +68,148 @@ test(`priotizes generators via extend`, () => {
 	expect(result.pets[0]?.breed).toBeTypeOf('string');
 	expect(result.totalVisits).toBeTypeOf('number');
 });
+
+test(`fixture has all the zod types`, () => {
+	enum Fruits {
+		Apple,
+		Banana,
+	}
+	enum StringFruits {
+		Apple = 'apple',
+		Banana = 'banana',
+		Cantaloupe = 3,
+	}
+	const constFruits = {
+		Apple: 'apple',
+		Banana: 'banana',
+		Cantaloupe: 3,
+	} as const;
+	const Person = z.object({
+		name: z.string(),
+	});
+
+	const Employee = z.object({
+		role: z.string(),
+	});
+
+	class InstanceOfClass {}
+
+	const instanceOfSchema = z.instanceof(InstanceOfClass);
+
+	const schemaWithEverything = z.object({
+		string: z.string(),
+		number: z.number(),
+		bigint: z.bigint(),
+		boolean: z.boolean(),
+		date: z.date(),
+		symbol: z.symbol(),
+		undefined: z.undefined(),
+		null: z.null(),
+		void: z.void(),
+		any: z.any(),
+		// TODO: add this?
+		// unknown: z.unknown(),
+		never: z.never(),
+		literal: z.literal('literal'),
+		string_max: z.string().max(5),
+		string_min: z.string().min(5),
+		string_length: z.string().length(5),
+		string_email: z.string().email(),
+		string_url: z.string().url(),
+		string_emoji: z.string().emoji(),
+		string_uuid: z.string().uuid(),
+		// this is by design
+		// we don't support cuid in favor of cuid2
+		// string_cuid: z.string().cuid()
+		// TODO: the seeding doesn't work for this?
+		string_cuid2: z.string().cuid2(),
```

Comment:
Hrmmm... the seeding should work.

```ts
  this.cuid2 = Cuid2({ random: this.unitInterval.bind(this) });
```

Maybe the binding isn't working as expected?

Reply:
I also tried something else, but it still failed.
Because of some lack of time I just left it as-is 😅